crates/bevy_render/src/extract_instances.rs

```diff
@@ -94,60 +94,60 @@ where
     }
 }
```

Comment:
@pcwalton / @superdump  I don't know whether there is something behind I'm missing (I'm quite a new in Rust). There is defined twice:

```
impl<EI> ExtractInstancesPlugin<EI>
where
    EI: ExtractInstance,
{
```

So it can be merged as one definition with two methods.

Am I correct?